src/content/getting-started/access-to-platform-api/_index.md

```diff
@@ -13,3 +13,106 @@ user_questions:
   - How can I access the platform API?
   - What do I need to do to access the platform API?
 ---
+
+Giant Swarm's platform API, a Kubernetes API that operates on the management cluster, offers unique features and custom resources. These enable you to perform a range of actions, from creating and updating clusters to deploying applications, enhancing your cluster and workload resource management capabilities.
```

Comment:
> And should we be very clear and say ", the Kubernetes API of the management cluster,"? So that customers don't accidentally think that our API is something homegrown and separate.

I think the first phrase states that